### ISSUE TITLE
Material UI theme provokes FTA-FMEA to crash

### ISSUE THREAD

**Issue by LaChope** [kbss-cvut/fta-fmea-ui#305]
@blcham @Kasmadei 
On both the development server and dockerized version, app crashes with: 

**Firefox error:**
![Image](https://github.com/kbss-cvut/fta-fmea-ui/assets/57750426/a229a438-dd45-4d20-8045-5c97c266a8a2)

**Chrome error:**
![Image](https://github.com/kbss-cvut/fta-fmea-ui/assets/57750426/35fcf4c2-063d-42e3-8971-44821eb4c75b)

The issue comes very probably from the changes of [this commit](https://github.com/kbss-cvut/fta-fmea-ui/commit/526f0525b9772c3e30ed7237049c5f55b0e4b092#diff-eca6c6dd464d1cab34bbd9560d4120bc04da1e34b64dbdf0f1dab58333d2e557).

**Comment by Kasmadei** [comment]
@LaChope Hi, i am already fixed it here: 
https://github.com/kbss-cvut/fta-fmea-ui/pull/304

**Comment by LaChope** [comment]
Reopening as not completed (see comment just above)

**Comment by blcham** [comment]
It might be little relevant to: https://github.com/kbss-cvut/record-manager-ui/issues/125#issuecomment-2077283029

**Comment by blcham** [comment]
@LaChope what do you propose to do with this? Should we put it to TODO ?

**Comment by LaChope** [comment]
Yes, also I am updating the description as #365 should be completed before this.

**Comment by blcham** [comment]
I guess this is not relevant for the delivery -- putting it into "technical debt sprint"

**Comment by LaChope** [comment]
@blcham yes
